crds/examples/AzureApplicationGatewayLoadDistributionPolicy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: appgw.ingress.azure.io/v1beta1
+kind: AzureApplicationGatewayLoadDistribitionPolicy
+metadata:
+  name: load-distribution-policy-with-failover
```

Comment:
Does it have to be active-passive pair?

Reply:
it does not have to be. someone can have all active servers if they would like. services are grouped by what LDP they belong to